{"name": "st.family -- @1.1 transfer types", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: '/utiles/bot_file.xls'\n", "trace": "  File \"c:\\users\\sasa\\anaconda3\\envs\\finbot\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\sasa\\anaconda3\\envs\\finbot\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\charity.py\", line 8, in step_impl\n    assert client.check_message_code(code)\n  File \"C:\\Users\\sasa\\PycharmProjects\\FinBot\\base\\user_telegram.py\", line 107, in check_message_code\n    self.client.loop.run_until_complete(self.async_check_message_code(code))\n  File \"c:\\users\\sasa\\anaconda3\\envs\\finbot\\lib\\asyncio\\base_events.py\", line 587, in run_until_complete\n    return future.result()\n  File \"C:\\Users\\sasa\\PycharmProjects\\FinBot\\base\\user_telegram.py\", line 81, in async_check_message_code\n    search_text = find_text_from_table(int(code))\n  File \"C:\\Users\\sasa\\PycharmProjects\\FinBot\\base\\functions.py\", line 8, in find_text_from_table\n    table = pyexcel.get_book_dict(file_name='/utiles/bot_file.xls')\n  File \"c:\\users\\sasa\\anaconda3\\envs\\finbot\\lib\\site-packages\\pyexcel\\core.py\", line 235, in get_book_dict\n    book = get_book(**keywords)\n  File \"c:\\users\\sasa\\anaconda3\\envs\\finbot\\lib\\site-packages\\pyexcel\\core.py\", line 47, in get_book\n    book_stream = sources.get_book_stream(**keywords)\n  File \"c:\\users\\sasa\\anaconda3\\envs\\finbot\\lib\\site-packages\\pyexcel\\internal\\core.py\", line 39, in get_book_stream\n    sheets = a_source.get_data()\n  File \"c:\\users\\sasa\\anaconda3\\envs\\finbot\\lib\\site-packages\\pyexcel\\plugins\\sources\\file_input.py\", line 38, in get_data\n    sheets = self.__parser.parse_file(self.__file_name, **self._keywords)\n  File \"c:\\users\\sasa\\anaconda3\\envs\\finbot\\lib\\site-packages\\pyexcel\\plugins\\parsers\\excel.py\", line 19, in parse_file\n    return self._parse_any(file_name, **keywords)\n  File \"c:\\users\\sasa\\anaconda3\\envs\\finbot\\lib\\site-packages\\pyexcel\\plugins\\parsers\\excel.py\", line 40, in _parse_any\n    sheets = get_data(anything, file_type=file_type, **keywords)\n  File \"c:\\users\\sasa\\anaconda3\\envs\\finbot\\lib\\site-packages\\pyexcel_io\\io.py\", line 87, in get_data\n    afile, file_type=file_type, streaming=False, **keywords\n  File \"c:\\users\\sasa\\anaconda3\\envs\\finbot\\lib\\site-packages\\pyexcel_io\\io.py\", line 105, in _get_data\n    return load_data(**keywords)\n  File \"c:\\users\\sasa\\anaconda3\\envs\\finbot\\lib\\site-packages\\pyexcel_io\\io.py\", line 191, in load_data\n    reader.open(file_name, **keywords)\n  File \"c:\\users\\sasa\\anaconda3\\envs\\finbot\\lib\\site-packages\\pyexcel_io\\reader.py\", line 48, in open\n    file_name, self.file_type, **native_sheet_keywords\n  File \"c:\\users\\sasa\\anaconda3\\envs\\finbot\\lib\\site-packages\\pyexcel_xls\\xlsr.py\", line 181, in __init__\n    super().__init__(file_type, filename=file_name, **keywords)\n  File \"c:\\users\\sasa\\anaconda3\\envs\\finbot\\lib\\site-packages\\pyexcel_xls\\xlsr.py\", line 146, in __init__\n    self.xls_book = self.get_xls_book(**xlrd_params)\n  File \"c:\\users\\sasa\\anaconda3\\envs\\finbot\\lib\\site-packages\\pyexcel_xls\\xlsr.py\", line 167, in get_xls_book\n    xls_book = xlrd.open_workbook(**xlrd_params)\n  File \"c:\\users\\sasa\\anaconda3\\envs\\finbot\\lib\\site-packages\\xlrd\\__init__.py\", line 111, in open_workbook\n    with open(filename, \"rb\") as f:\n"}, "steps": [{"name": "Given client send message -/start-", "status": "passed", "start": 1605247098454, "stop": 1605247101181}, {"name": "And client click the button -Благодійність-", "status": "passed", "start": 1605247101181, "stop": 1605247103762}, {"name": "And client click the button -ST.FAMILY-", "status": "passed", "start": 1605247103762, "stop": 1605247106300}, {"name": "And client send message -запит-", "status": "passed", "start": 1605247106300, "stop": 1605247110704}, {"name": "And client click the button -Відмінити тікет-", "status": "passed", "start": 1605247110704, "stop": 1605247113271}, {"name": "When client click the button -Так-", "status": "passed", "start": 1605247113271, "stop": 1605247116753}, {"name": "Then client check message -Тікет скасований-1138-", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: '/utiles/bot_file.xls'\n", "trace": "  File \"c:\\users\\sasa\\anaconda3\\envs\\finbot\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\sasa\\anaconda3\\envs\\finbot\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\charity.py\", line 8, in step_impl\n    assert client.check_message_code(code)\n  File \"C:\\Users\\sasa\\PycharmProjects\\FinBot\\base\\user_telegram.py\", line 107, in check_message_code\n    self.client.loop.run_until_complete(self.async_check_message_code(code))\n  File \"c:\\users\\sasa\\anaconda3\\envs\\finbot\\lib\\asyncio\\base_events.py\", line 587, in run_until_complete\n    return future.result()\n  File \"C:\\Users\\sasa\\PycharmProjects\\FinBot\\base\\user_telegram.py\", line 81, in async_check_message_code\n    search_text = find_text_from_table(int(code))\n  File \"C:\\Users\\sasa\\PycharmProjects\\FinBot\\base\\functions.py\", line 8, in find_text_from_table\n    table = pyexcel.get_book_dict(file_name='/utiles/bot_file.xls')\n  File \"c:\\users\\sasa\\anaconda3\\envs\\finbot\\lib\\site-packages\\pyexcel\\core.py\", line 235, in get_book_dict\n    book = get_book(**keywords)\n  File \"c:\\users\\sasa\\anaconda3\\envs\\finbot\\lib\\site-packages\\pyexcel\\core.py\", line 47, in get_book\n    book_stream = sources.get_book_stream(**keywords)\n  File \"c:\\users\\sasa\\anaconda3\\envs\\finbot\\lib\\site-packages\\pyexcel\\internal\\core.py\", line 39, in get_book_stream\n    sheets = a_source.get_data()\n  File \"c:\\users\\sasa\\anaconda3\\envs\\finbot\\lib\\site-packages\\pyexcel\\plugins\\sources\\file_input.py\", line 38, in get_data\n    sheets = self.__parser.parse_file(self.__file_name, **self._keywords)\n  File \"c:\\users\\sasa\\anaconda3\\envs\\finbot\\lib\\site-packages\\pyexcel\\plugins\\parsers\\excel.py\", line 19, in parse_file\n    return self._parse_any(file_name, **keywords)\n  File \"c:\\users\\sasa\\anaconda3\\envs\\finbot\\lib\\site-packages\\pyexcel\\plugins\\parsers\\excel.py\", line 40, in _parse_any\n    sheets = get_data(anything, file_type=file_type, **keywords)\n  File \"c:\\users\\sasa\\anaconda3\\envs\\finbot\\lib\\site-packages\\pyexcel_io\\io.py\", line 87, in get_data\n    afile, file_type=file_type, streaming=False, **keywords\n  File \"c:\\users\\sasa\\anaconda3\\envs\\finbot\\lib\\site-packages\\pyexcel_io\\io.py\", line 105, in _get_data\n    return load_data(**keywords)\n  File \"c:\\users\\sasa\\anaconda3\\envs\\finbot\\lib\\site-packages\\pyexcel_io\\io.py\", line 191, in load_data\n    reader.open(file_name, **keywords)\n  File \"c:\\users\\sasa\\anaconda3\\envs\\finbot\\lib\\site-packages\\pyexcel_io\\reader.py\", line 48, in open\n    file_name, self.file_type, **native_sheet_keywords\n  File \"c:\\users\\sasa\\anaconda3\\envs\\finbot\\lib\\site-packages\\pyexcel_xls\\xlsr.py\", line 181, in __init__\n    super().__init__(file_type, filename=file_name, **keywords)\n  File \"c:\\users\\sasa\\anaconda3\\envs\\finbot\\lib\\site-packages\\pyexcel_xls\\xlsr.py\", line 146, in __init__\n    self.xls_book = self.get_xls_book(**xlrd_params)\n  File \"c:\\users\\sasa\\anaconda3\\envs\\finbot\\lib\\site-packages\\pyexcel_xls\\xlsr.py\", line 167, in get_xls_book\n    xls_book = xlrd.open_workbook(**xlrd_params)\n  File \"c:\\users\\sasa\\anaconda3\\envs\\finbot\\lib\\site-packages\\xlrd\\__init__.py\", line 111, in open_workbook\n    with open(filename, \"rb\") as f:\n"}, "start": 1605247116753, "stop": 1605247117050}], "parameters": [{"name": "button", "value": "ST.FAMILY"}], "start": 1605247098454, "stop": 1605247117050, "uuid": "a0033c52-833c-476f-af85-8f3bc61cc3db", "historyId": "59b4d465c53b2a663266ce74295d9132", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "sd_test12_bot"}, {"name": "feature", "value": "finbot_spd_charity"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}